src/dataset.py

```diff
@@ -28,7 +29,7 @@
 client = Socrata("data.seattle.gov", None)
 LICENSE_DATASET = "enxu-fgzb"
 SALARY_DATASET = "2khk-5ukd"
-DATASET_PATH = Path(__file__).absolute().parent / "data" / "1-312-data.db"
+DATA_API_HOST = "https://spd-lookup.herokuapp.com"
```

Comment:
Now that we have the API accessible on the new server, this can be changed!
```suggestion
DATA_API_HOST = "https://1312api.tech-bloc-sea.dev"
```